QUARKUS-5623.md

```diff
@@ -0,0 +1,36 @@
+# QUARKUS-5623 - Logging in OpenTelemetry
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-5623
+
+Upstream PRs:
+    - https://github.com/quarkusio/quarkus/pull/38239
+    - https://github.com/quarkusio/quarkus/pull/46354
+
+Documentation: https://quarkus.io/version/main/guides/opentelemetry-logging
+
+## Preview status
+OpenTelemetry logging is currently considered dev-preview in quarkus.
+
+## Scope of testing
+Test OpenTelemetry quarkus extension.
+Verify that quarkus application is sending its logs to the Grafana backend.
```

Comment:
`Grafana backend` ... what exactly do you mean by that. To me Grafana is solution for dashboards, it takes data from a "data source" - https://grafana.com/docs/grafana/latest/datasources/#built-in-core-data-sources

Reply:
OK, I will clarify this more in the TP. I intend to use grafana/otel-lgtm image, which is basically kind of all-in-one for grafana (display), Loki (logging itself) and more.

Reply:
I've added description to the TP, I hope it is sufficient.